Bottom: 758305d55504b97fc2c3acada99fcfc886fc35bc
Top:    6902559dcceb2ce2f27e3ac063703bcfbb6d5fcf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-02 21:38:05 +0100

Add rule to extract localized stings from .res files


---

diff --git a/autogen.sh b/autogen.sh
index d1671fb..3eeeca2 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -70,6 +70,12 @@ sed \
 	-e 's/^GENPOT *=/GENPOT = XGETTEXT=\${XGETTEXT} MSGFMT=\${MSGFMT} /g' \
 	-e 's/ChangeLog//g' \
 	po/Makefile.in.in.orig > po/Makefile.in.in
+
+# Add rule for .res file processing
+echo "
+%.res.h: %.res
+	\$(INTLTOOL_EXTRACT) --type=gettext/quoted $^" >> po/Makefile.in.in
+
 rm -f po/Makefile.in.in.orig
 
 ############################################################################
